website/views/options/settings/subscription.jade

```diff
@@ -49,7 +49,9 @@ script(id='partials/options.settings.subscription.html',type='text/ng-template',
 
         div(ng-if='user.purchased.plan.customerId')
           .btn.btn-primary(ng-if='!user.purchased.plan.dateTerminated && user.purchased.plan.paymentMethod=="Stripe"', ng-click='Payments.showStripeEdit()')=env.t('subUpdateCard')
-          .btn.btn-sm.btn-danger(ng-if='!user.purchased.plan.dateTerminated && user.purchased.plan.customerId !== "group-plan"', ng-click='Payments.cancelSubscription()')=env.t('cancelSub')
+          .btn.btn-sm.btn-danger(ng-if='user.purchased.plan.paymentMethod!="Apple" && user.purchased.plan.paymentMethod!="Google" && !user.purchased.plan.dateTerminated && user.purchased.plan.customerId !== "group-plan"', ng-click='Payments.cancelSubscription()')=env.t('cancelSub')
```

Comment:
I was just create the constants in the Payments service on the client. Yes, we would like to de duplicate code from the server and the client (and we do have a common folder), but a better solution would be to use bundling differently.